{"uuid":"17851f2a-5dd2-49c9-9df4-d80095b662ea","name":"Scenario Outline: Verify that user receives error for invalid python code for Max Consecutive Ones Question","children":["f7f500fe-4541-4154-b697-9599eb1543dc"],"befores":[{"name":"hooks.HooksPage.setUp()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747077330620,"stop":1747077332721}],"afters":[{"name":"hooks.HooksPage.tearDown(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1747077335686,"stop":1747077335820}],"start":1747077330620,"stop":1747077335820}